dashboard/app/models/blockly.rb

```diff
@@ -329,7 +329,7 @@ def localized_authored_hints
         translated_text = translations.try(:[], hint['hint_id'].to_sym)
         original_text = hint['hint_markdown']
 
-        if translated_text != original_text
+        unless translated_text.nil? || translated_text == original_text
```

Comment:
LGTM. Though I think `if !translated_text.nil? && translated_text != original_text` is easier to read, I spent longer than I feel I should have to understand this.